services/QuillLMS/engines/comprehension/app/models/comprehension/rule.rb

```diff
@@ -98,5 +114,64 @@ def display_name
         errors.add(:prompts, "prompt #{prompt.id} already has a plagiarism rule") if prompt.rules.where(rule_type: TYPE_PLAGIARISM).first&.id
       end
     end
+
+    def log_creation(user_id)
+      if regex?
+        send_change_log(user_id, :create_regex, change_text_regex)
+      elsif plagiarism?
+        send_change_log(user_id, :create_plagiarism, change_text_plagiarism)
+      elsif universal?
+        send_change_log(user_id, :create_universal, change_text_universal)
+      end
+    end
+
+    def log_deletion(user_id)
+      if regex?
+        send_change_log(user_id, :delete_regex, nil, change_text_regex)
```

Comment:
Since this is a one liner, I'd do it with the `if` at the end of the `send_change_log` line.